backend/src/lib/emailVerificationPlugin.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose')
+const nanoid = require('nanoid').customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 40)
+const { sendEmailVerification } = require('../email')
+
+const EmailToken = mongoose.model(
+  'EmailToken',
+  {
+    token: {
+      type: String,
+      default: nanoid(),
+    },
+    user: {
+      type: mongoose.Types.ObjectId,
+      required: true,
+    },
+    createdAt: { type: Date, default: Date.now },
```

Comment:
Should add expired.

On the login form, we need a `send email again`  functionality

Reply:
👌